docs/05-technical-reference/00-custom-resources/ac-01-application.md

```diff
@@ -40,12 +40,9 @@ This table lists all the possible parameters of a given resource together with t
 |----------|:-------------:|------|
 | **metadata.name** | Yes | Specifies the name of the CR. |
 | **spec.description** | No | Describes the connected Application.  |
-| **spec.accessLabel** | No | Labels the Application when an ApplicationMapping is created. |
```

Comment:
I removed that 'cause it references Service Catalog. Should we replace it with anything or is it all good?